File: solutions/beecrowd/1008/1008.pas
Uses sysutils;

Var a : Integer;
    b : Integer;
    c : Double;

begin
    read(a);
    read(b);
    read(c);

    WriteLn(Format('NUMBER = %d', [a]));
    WriteLn(Format('SALARY = U$ %.2f', [b * c]));
end.

File: solutions/beecrowd/1008/1008.exs
a = IO.gets("") |> String.trim |> String.to_integer
b = IO.gets("") |> String.trim |> String.to_integer
c = IO.gets("") |> String.trim |> String.to_float

IO.puts "NUMBER = #{a}"
IO.puts "SALARY = U$ #{:io_lib.format("~.2f", [c * b])}"
